AggregateMetrics/AggregateMetrics/AzureWebApp/Implementation/SumUpGauge.cs

```diff
@@ -42,7 +42,7 @@ public MetricTelemetry GetValueAndReset()
 
             metric.Name = this.name;
             metric.Value = this.gaugesToSum.Sum((g) => { return g.GetValueAndReset().Value; });
-            metric.Context.GetInternalContext().SdkVersion = new PerformanceCollectorModule().sdkVersion;
+            metric.Context.GetInternalContext().SdkVersion = SdkVersionAzureWebApp.VersionPrefix + SdkVersionAzureWebApp.GetAssemblyVersion();
```

Comment:
since you are doing it every time metric got created I'd recommend to precalculate this string in static variable instead of constructing it every time from scratch